package.json

```diff
@@ -25,6 +25,7 @@
     "csv-parse": "^1.1.7",
     "express": "^4.14.0",
     "node-postal": "^0.3.0",
+    "pbf2json": "^4.2.0",
```

Comment:
I like the idea of using the binaries included in the npm module.

I'm not sure why I decided to clone the source repo in the Dockerfile, we could probably get rid of that now.